src/VolumeCache.ts

```diff
@@ -222,15 +218,15 @@ export default class VolumeCache {
 
     // Check if entry is already in cache
     for (const existingEntry of entryList) {
-      if (extentIsEqual(existingEntry.extent, extent)) {
+      if (existingEntry.region.equals(region)) {
```

Comment:
I just thought of something related to two regions being "equal".  Do you consider two regions equal if they result in the same pixels?  Because that allows for some rounding off error with float values that land in between exact integer pixels.

Reply:
For the moment, `VolumeCache` only works in pixels anyways - it should never have to scale from ratios to pixels. (The lack of clarity about when we're working in which space is potentially a pitfall of a unified type. Perhaps I should go over both PRs and make sure the documentation is clear when we expect which?)

Reply:
it definitely helps when there's a comment saying what coordinate space your numbers are supposed to be in.